index.js

```diff
@@ -0,0 +1,10 @@
+var newman = require('newman'); // require Newman in your project
```

Comment:
czy to jest potrzebne?

Reply:
Nie, ale myślałem, że w ten sposób mamy wykonać skrypt. Mogę usunąć.